bpmn/Dockerfile

```diff
@@ -19,7 +20,7 @@ WORKDIR /app
 RUN PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1 npm ci
 RUN npm run lint
 
-RUN /usr/local/bin/pkg --targets ${NODE}-${PLATFORM}-$([ "$TARGETARCH" == "amd64" ] && echo "x64" || echo "$TARGETARCH") . -o app.bin
```

Comment:
We need to keep `alpine` as "platform". `pkg` makes a distinction between linux and alpine:

> **platform** alpine, linux, linuxstatic, win, macos, (freebsd)

https://github.com/vercel/pkg#targets